frontend/gateway/pb/gateway.proto

```diff
@@ -63,6 +64,36 @@ message RefMap {
 	map<string, Ref> refs = 1;
 }
 
+message Attestations {
+	message Attestation {
+		oneof Attestation {
+			InToto intoto = 1;
+		}
+	}
+	repeated Attestation attestation = 1;
+}
+
+message InToto {
+	string predicateType = 1;
+	Ref predicateRef = 2;
+	string predicatePath = 3;
```

Comment:
Follow-up: as discussed for SBOMs we should have a way that this path can point to a list of attestations. So SBOM generator can generate multiple SBOMs with a single invocation that are listed in a file and BuildKit will create an independent attestation record for each file.